app/controllers/hyrax/dashboard/collections_controller.rb

```diff
@@ -168,10 +168,10 @@ def after_destroy(_id)
         # leaving id to avoid changing the method's parameters prior to release
         respond_to do |format|
           format.html do
-            redirect_to my_collections_path,
+            redirect_to hyrax.my_collections_path,
```

Comment:
Without this change the controller test that I turned back on would redirect to: `http://test.host/assets?action=index&controller=hyrax%2Fmy%2Fcollections&locale=en`
I'm not sure why the analogous controller test for the non-valkyrie based controller doesn't have the same problem, but this does fix the problem.